flocker/node/agents/test/test_blockdevice.py

```diff
@@ -509,9 +525,30 @@ def test_unrelated_mounted(self):
         dataset returned by ``BlockDeviceDeployer.discover_state`` and not as a
         manifestation on the ``NodeState``.
         """
-        unrelated_device = FilePath(self.mktemp())
-        with unrelated_device.open("w") as unrelated_file:
-            unrelated_file.truncate(LOOPBACK_MINIMUM_ALLOCATABLE_SIZE)
+        # Create a different one so that the filesystem we get from it isn't
+        # accidentally treated specially by the API object we're going to let
+        # the deployer use.  In other words, this use of LoopbackBlockDeviceAPI
```

Comment:
:question: Suggested rephrasing:

```
# Create a different API so that the filesystem we get from it isn't
# accidentally treated specially by the API under test. 
```

i.e.
* s/one/API/
* fewer words at end of sentence.